### PR TITLE
Implement progress bar for conduct load

### PR DESCRIPTION
**Pull request by fsat**

### CODE REVIEW

**Comment by huntc** [comment]
LGTM

File: conductr_cli/resolvers/uri_resolver.py [hunk 1]
```diff
@@ -60,3 +60,49 @@ def cache_path(cache_dir, uri):
...
+            return '#'
+        else:
+            return ' '
+
+    progress = ''.join([progress_character(i) for i in range(1, bar_range)])
+    return '[{}] {}'.format(progress, '%3d%%' % percent)
+
+
+def print_replace(message):
+    flush_character = ''.join(['\b' for c in message])
```
Comment (ktoso) [inline conductr_cli/resolvers/uri_resolver.py]: Interesting, why not a single `\r`?
This is message.length backspace chars right?

File: conductr_cli/logging_setup.py [hunk 2]
```diff
@@ -36,6 +55,37 @@ def screen(self, message, *args, **kwargs):
...
+
+    # continue as normal
+    log.info('Hey')
+    ```
+    """
+    flush_required = kwargs.pop('flush')
+
+    line_end = '\r'
+    if flush_required:
+        line_end = '\n'
```
Comment (fsat) [inline conductr_cli/logging_setup.py:84]: @ktoso - replaced multiple backspace character with `\r`; much better - thank you!

**Comment by fsat** [comment]
Manual test done successfully

Test step:
- Clear cache
- Execute `conduct load reactive-maps-frontend` (which resolves and downloads from bintray)

Progress bar should be visible.

Repeat the steps above with `conduct load -q`, with quiet mode progress bar should not be visible, and it displays `bundleId` of the newly loaded bundle.

```
Felixs-MBP:conductr-cli felixsatyaputra$ rm -rf ~/.conductr/cache/reactive-maps-frontend-v1-023f9da2243a0751c2e231b452aa3ed32fbc35351c543fbd536eea7ec457cfe2.zip && python3 -m conductr_cli.conduct load reactive-maps-frontend
Retrieving bundle...
Loading bundle from cache typesafe/bundle/reactive-maps-frontend
Retrieving file:///Users/felixsatyaputra/workspace/typesafe-fsat/conductr-cli/reactive-maps-frontend
Resolving bundle typesafe/bundle/reactive-maps-frontend
Retrieving https://dl.bintray.com/typesafe/bundle/typesafe/reactive-maps-frontend/v1-023f9da2243a0751c2e231b452aa3ed32fbc35351c543fbd536eea7ec457cfe2/reactive-maps-frontend-v1-023f9da2243a0751c2e231b452aa3ed32fbc35351c543fbd536eea7ec457cfe2.zip
[#################################################] 100%
Loading bundle to ConductR...
Bundle 023f9da2243a0751c2e231b452aa3ed3 is installed
Bundle loaded.
Start bundle with: conduct run --ip 127.0.0.1 023f9da
Unload bundle with: conduct unload --ip 127.0.0.1 023f9da
Print ConductR info with: conduct info --ip 127.0.0.1

Felixs-MBP:conductr-cli felixsatyaputra$ rm -rf ~/.conductr/cache/reactive-maps-frontend-v1-023f9da2243a0751c2e231b452aa3ed32fbc35351c543fbd536eea7ec457cfe2.zip && python3 -m conductr_cli.conduct load -q reactive-maps-frontend
023f9da2243a0751c2e231b452aa3ed3
```

**Comment by fsat** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;Qg2RbhGD&#x2F;177-add-progress-bar-for-cli-downloads"> Add progress bar for CLI downloads</a>

File: conductr_cli/logging_setup.py (hunk 2 above)
Comment (ktoso) [inline conductr_cli/logging_setup.py:84]: :+1:  :-)

**Pull request merged**